requirements/test.txt

```diff
@@ -18,3 +18,4 @@ path.py>=12.4
 ddt
 nose
 coverage
+setuptools-scm
```

Comment:
This addition should not be necessary, now that we've added `setuptools-scm` to `base.txt` and we're already saying `-r base.txt` at the top of this file.

Reply:
we only have `setuptools-scm` in `setup.txt` at the moment.

Reply:
I moved the `setuptools-scm` to `base.txt` now and removed from here.